pynicotine/gtkgui/ui/about/about.ui

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk+" version="3.18"/>
+  <object class="GtkAboutDialog" id="About">
+    <property name="can_focus">False</property>
+    <property name="title" translatable="yes">About Nicotine+</property>
+    <property name="modal">True</property>
+    <property name="window_position">center-on-parent</property>
+    <property name="logo_icon_name">org.nicotine_plus.Nicotine</property>
+    <property name="program_name" translatable="yes">Nicotine+</property>
+    <property name="comments" translatable="yes">A graphical client for the Soulseek peer-to-peer system
+Based on code from Nicotine and PySoulSeek
+
+This product includes IP2Location LITE data, available from:
+https://www.ip2location.com</property>
+    <property name="website">https://nicotine-plus.org</property>
+    <property name="license_type">GTK_LICENSE_GPL_3_0_ONLY</property>
+    <property name="authors"># Nicotine+ MAINTAINERS
+
+### Active
```

Comment:
@mathiascode, can you have this loaded at runtime from disk so we don't have redundant copies of this text?

Reply:
I had to revert this change. The dialog elements are destroyed when you close the dialogs.